HISTORY.rst

```diff
@@ -23,6 +23,8 @@ Next Release
 * Add tests for detecting gene annotations (and verifying they are in
   MIRIAM style)
 * Add unit tests for ``matrix.py`` in file ``test_for_matrix.py``.
+* Add test to detect infeasible metabolites (i.e. metabolites are cannot
```

Comment:
I think calling these metabolites `infeasible metabolites` makes it a little cryptic. I'd like you to stick with the established naming pattern, despite it being longer i.e.
`find_metabolites_produced_with_closed_bounds`
`find_metabolites_consumed_with_closed_bounds`
hence:
`find_metabolites_not_produced_with_open_bounds`
`find_metabolites_not_consumed_with_open_bounds`